science/gildas/Portfile

```diff
@@ -59,7 +62,7 @@ require_active_variants gtk2 x11 quartz
 compilers.enforce_some_fortran cfitsio
 
 compilers.choose    fc
-compilers.setup     -g95 -gcc44 -gcc45 -gcc46 require_fortran
+compilers.setup     -gcc44 -gcc45 -gcc46 require_fortran
```

Comment:
Was this change intentional? It doesn't look like it's related to the sort issue.

Reply:
Yes it was intentional. Just cleaned this obsolete stuff by going through the Portfile. No, it is not directly related to the issue. Should I undo this part?

Reply:
Since it's unrelated, I'd put it in a separate commit so that you have an opportunity to describe the change separately. It's fine if the second commit is also in this PR.